plugins/CorePluginsAdmin/vue/src/PluginsTable/PluginsTable.vue

```diff
@@ -380,6 +380,15 @@ export default defineComponent({
         redirectTo: 'referrer',
       })}`;
     },
+    isMatomoUrl(url: string) {
+      let isMatomoUrl = false;
+      this.matomoUrls.forEach((matomoUrl: string) => {
```

Comment:
Can we replace this with a simple check on the exact hostname?

```javascript
  try {
    const pluginHost = (new URL(url)).host;

    return this.matomoHosts.indexOf(pluginHost) !== -1;
  } catch(error) {
    // the plugin may provide a broken/invalid url
    return false;
  }
```

Using a prefix for matching allows for some weird things like `http://www.matomo.org.some.other.domain.com`. And the configuration of both `...org` and `...org/` is unnecessary because the one without `/` will already match.

If we also ignore the protocol (should not provide much value here), we are able to trim the complete check down to the 4 relevant domains :thinking: